rwengine/src/render/GameRenderer.cpp

```diff
@@ -235,15 +235,15 @@ void GameRenderer::renderWorld(GameWorld* world, const ViewCamera& camera,
 
     culled = 0;
 
-    renderObjects(world);
-
-    renderer->pushDebugGroup("Water");
+	renderer->pushDebugGroup("Water");
```

Comment:
You are breaking here code formatting. (4 spaces should be used instead tab)

Also (new) in line 240, 246.

Reply:
Fixed tabs mixing.